packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -587,7 +587,7 @@ class DragTargetDetails<T> {
   DragTargetDetails({required this.data, required this.offset});
 
   /// The data that was dropped onto this [DragTarget].
-  final T data;
+  final T? data;
```

Comment:
Again, this has the potential to be super super breaking. *Any* customer that uses drag target details now has to update their code to handle if this is null, as none of them will have that now.